math/benches/elliptic_curves/bn_254.rs

```diff
@@ -28,11 +28,15 @@ pub fn bn_254_elliptic_curve_benchmarks(c: &mut Criterion) {
     let mut rng = StdRng::seed_from_u64(42);
     let a_val: u128 = rng.gen();
     let b_val: u128 = rng.gen();
-    let a_g1 = BN254Curve::generator().operate_with_self(a_val);
-    let b_g1 = BN254Curve::generator().operate_with_self(b_val);
-
-    let a_g2 = BN254TwistCurve::generator().operate_with_self(a_val);
-    let b_g2 = BN254TwistCurve::generator().operate_with_self(b_val);
+    let a_g1 = BN254Curve::generator().operate_with_self(a_val).to_affine();
+    let b_g1 = BN254Curve::generator().operate_with_self(b_val).to_affine();
+
+    let a_g2 = BN254TwistCurve::generator()
+        .operate_with_self(b_val)
+        .to_affine();
+    let b_g2 = BN254TwistCurve::generator()
+        .operate_with_self(b_val)
+        .to_affine();
```

Comment:
I believe this changes the whole meaning of the benchmark. E.g., the first two bench runs are precisely for converting to affine, which becomes trivial with the change.